development/circular-deps.ts

```diff
@@ -20,42 +30,7 @@ const FILE_HEADER = `// This is a machine-generated file that tracks circular de
  * Message displayed when circular dependency checks fail and need resolution.
  */
 const RESOLUTION_STEPS =
-  'To resolve this issue, run `yarn circular-deps:update` locally and commit the changes.';
-
-/**
- * Patterns for files and directories to ignore when checking for circular dependencies:
- * - test files and directories
- * - storybook files and directories
- * - any file with .test., .spec., or .stories. in its name
- */
-const IGNORE_PATTERNS = [
-  // Test files and directories
-  '**/test/**',
-  '**/tests/**',
-  '**/*.test.*',
-  '**/*.spec.*',
-
-  // Storybook files and directories
-  '**/stories/**',
-  '**/storybook/**',
-  '**/*.stories.*',
-];
-
-/**
- * Source code directories to check for circular dependencies.
- * These are the main app directories containing production code.
- */
-const ENTRYPOINT_PATTERNS = [
-  'app/**/*', // Main application code
-  'shared/**/*', // Shared utilities and components
-  'ui/**/*', // UI components and styles
-];
```

Comment:
`ENTRYPOINT_PATTERNS` are just directories now, not globs, with no filters (`IGNORE_PATTERNS`), as they aren't needed.